src/operator/quantization/quantize_v2.cc

```diff
@@ -19,7 +19,7 @@
 
 /*!
  *  Copyright (c) 2017 by Contributors
```

Comment:
These copyright lines should not be in ASF-licensed files.

Reply:
done